src/databricks/labs/ucx/source_code/files.py

```diff
@@ -111,13 +112,11 @@ def __repr__(self):
 
 class LocalFileResolver(BaseImportResolver, BaseFileResolver):
```

Comment:
```suggestion
class ImportResolver(BaseFileResolver):
```
and get rid of BaseImportResolver.

Reply:
renamed to ImportFileResolver